Tutorials/pinpull.md

```diff
@@ -0,0 +1,49 @@
+# Pull- Pins
+
+Suppose a pin is configured as an input. If nothing is connected to the pin and the program tries to read the state of the pin, it would be in a 'floating' state i.e an unknown state. To prevent this, a pull-up or a pull-down state is defined. They are often used in the case of Buttons and Switches.
```

Comment:
Hyphenated is now used in the text as per wikipedia (https://en.wikipedia.org/wiki/Pull-up_resistor) but non-hyphenated is the default arg value in the firmware, so we can't change it in the text.